src/System.Net.Http/src/uap/System/Net/HttpHandlerToFilter.cs

```diff
@@ -26,65 +26,207 @@ namespace System.Net.Http
 {
     internal class HttpHandlerToFilter : HttpMessageHandler
     {
-        private readonly RTHttpBaseProtocolFilter _next;
+        private readonly RTHttpBaseProtocolFilter _filter;
+        private readonly RTHttpBaseProtocolFilter _filterWithNoCredentials;
         private int _filterMaxVersionSet;
+        private HttpClientHandler _handler;
 
-        internal HttpHandlerToFilter(RTHttpBaseProtocolFilter filter)
+        internal HttpHandlerToFilter(
+            RTHttpBaseProtocolFilter filter,
+            RTHttpBaseProtocolFilter filterWithNoCredentials,
+            HttpClientHandler handler)
         {
             if (filter == null)
```

Comment:
Good point.  But now that I am using Lazy pattern, there no longer is the filterWithNoCredentials in the handler object.